diff --git a/devtrac.install b/devtrac.install
index d067dfe..b2af5ed 100644
--- a/devtrac.install
+++ b/devtrac.install
@@ -426,19 +426,6 @@ function devtrac_install_tasks(&$install_state) {
     'display_name' => st('Configure Date Format'),
     'type' => 'form',
   );
-  $statistics_is_piwik = !empty($install_state['parameters']['stats']) && $install_state['parameters']['stats'] == 'piwik';
-  $statistics_is_googleanalytics = !empty($install_state['parameters']['stats']) && $install_state['parameters']['stats'] == 'googleanalytics';
-  $tasks['devtrac_install_statistics_select_form'] = array(
-    'display_name' => st('Enable site statistics'),
-    'type' => 'form',
-  );
-  $tasks['devtrac_install_statistics_config_form'] = array(
-    'display_name' => st('Configure site statistics'),
-    'type' => 'form',
-    'display' => $statistics_is_piwik || $statistics_is_googleanalytics,
-    'run' => $statistics_is_piwik || $statistics_is_googleanalytics ? INSTALL_TASK_RUN_IF_NOT_COMPLETED : INSTALL_TASK_SKIP,
-    'function' => $statistics_is_piwik ? 'devtrac_install_statistics_config_piwik_form' : 'devtrac_install_statistics_config_google_analytics_form',
-  );
   $tasks['devtrac_install_additional_modules'] = array(
     'display_name' => st('Install additional functionality'),
     'type' => 'batch',
@@ -572,140 +559,6 @@ function devtrac_install_welcome_submit($form, &$form_state) {
   $install_state['parameters']['welcome'] = 'done';
   $install_state['parameters']['locale'] = 'en';
 }
-
-/**
- * Task callback: uses Form API.
- */
-function devtrac_install_statistics_select_form() {
-  drupal_set_title(st('Enable site statistics'));
-
-  $options = array(
-    'none' => 'None',
-    'piwik' => t('Piwik'),
-    'googleanalytics' => t('Google Analytics'),
-  );
-  $form['module'] = array(
-    '#type' => 'radios',
-    '#title' => t('Site statistics module to use'),
-    '#default_value' => 'none',
-    '#options' => $options,
-    '#description' => t("The module to use to track your site's usage statistics."),
-  );
-
-  $form['actions'] = array('#type' => 'actions');
-  $form['actions']['submit'] = array(
-    '#type' => 'submit',
-    '#value' => st('Continue'),
-    '#weight' => 15,
-  );
-  return $form;
-}
-
-/**
- * Implements submit handler for devtrac_install_statistics_select_form.
- */
-function devtrac_install_statistics_select_form_submit($form, &$form_state) {
-  global $install_state;
-  if (isset($form_state['values']['module']) && ($form_state['values']['module'] != 'none')) {
-    $install_state['parameters']['stats'] = $form_state['values']['module'];
-  }
-}
-
-/**
- * Task callback: uses Form API.
- */
-function devtrac_install_statistics_config_piwik_form() {
-  drupal_set_title(st('Configure Piwik'));
-
-  $form['piwik_info'] = array(
-    '#title' => t('Information'),
-    '#prefix' => '<div>',
-    '#markup' => '<a href="www.piwik.org">Piwik - Web analytics</a> is an open source (GPL license) web analytics software. It gives interesting reports on your website visitors, your popular pages, the search engines keywords they used, the language they speak... and so much more. Piwik aims to be an open source alternative to Google Analytics.<br /><br />Please fill out the general settings. Piwik has many more settings. During this installation the defaults will be used. After the installation you can go to <a href="admin/config/system/piwik">admin/config/system/piwik</a> to change any of these settings.',
-    '#suffix' => '</div>',
-  );
-  $form['piwik_site_id'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Piwik site ID'),
-    '#default_value' => '',
-    '#required' => TRUE,
-    '#description' => t('The user account number is unique to the websites domain. Click the <strong>Settings</strong> link in your Piwik account, then the <strong>Websites</strong> tab and enter the appropriate site <strong>ID</strong> into this field.'),
-  );
-  $form['piwik_url_http'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Piwik HTTP URL'),
-    '#default_value' => '',
-    '#required' => TRUE,
-    '#description' => t('The URL to your Piwik base directory. Example: "http://www.example.com/piwik/".'),
-  );
-  $form['piwik_url_https'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Piwik HTTPS URL'),
-    '#default_value' => '',
-    '#description' => t('The URL to your Piwik base directory with SSL certificate installed. Required if you track a SSL enabled website. Example: "https://www.example.com/piwik/".'),
-  );
-
-  $form['actions'] = array('#type' => 'actions');
-  $form['actions']['submit'] = array(
-    '#type' => 'submit',
-    '#value' => st('Continue'),
-    '#weight' => 15,
-  );
-  return $form;
-}
-
-/**
- * Implements submit handler for devtrac_install_statistics_config_piwik_form.
- */
-function devtrac_install_statistics_config_piwik_form_submit($form, &$form_state) {
-  global $install_state;
-  $install_state['statistics']['piwik']['piwik_site_id'] = $form_state['values']['piwik_site_id'];
-  variable_set('piwik_site_id', $form_state['values']['piwik_site_id']);
-  $install_state['statistics']['piwik']['piwik_url_http'] = $form_state['values']['piwik_url_http'];
-  variable_set('piwik_url_http', $form_state['values']['piwik_url_http']);
-  if (isset($form_state['values']['piwik_url_https'])) {
-    $install_state['statistics']['piwik']['piwik_url_https'] = $form_state['values']['piwik_url_https'];
-    variable_set('piwik_url_https', $form_state['values']['piwik_url_https']);
-  }
-}
-
-/**
- * Task callback: uses Form API.
- */
-function devtrac_install_statistics_config_google_analytics_form() {
-  drupal_set_title(st('Configure Google Analytics'));
-
-  $form['googleanalytics_info'] = array(
-    '#title' => t('Information'),
-    '#prefix' => '<div>',
-    '#markup' => '<a href="www.google.com/analytics">Google Analytics</a> is a free (registration required) website traffic and marketing effectiveness service.<br /><br />Please fill out the general settings. Google Analytics has many more settings. During this installation the defaults will be used. After the installation you can go to <a href="admin/config/system/googleanalytics">admin/config/system/googleanalytics</a> to change any of these settings.',
-    '#suffix' => '</div>',
-  );
-  $form['googleanalytics_account'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Web Property ID'),
-    '#default_value' => 'UA-',
-    '#required' => TRUE,
-    '#description' => t('This ID is unique to each site you want to track separately, and is in the form of UA-xxxxxxx-yy. To get a Web Property ID, <a href="www.google.com/analytics">register your site with Google Analytics</a>, or if you already have registered your site, go to your Google Analytics Settings page to see the ID next to every site profile. <a href="https://developers.google.com/analytics/resources/concepts/gaConceptsAccounts#webProperty">Find more information in the documentation</a>.'),
-  );
-
-  $form['actions'] = array('#type' => 'actions');
-  $form['actions']['submit'] = array(
-    '#type' => 'submit',
-    '#value' => st('Continue'),
-    '#weight' => 15,
-  );
-  return $form;
-}
-
-/**
- * Implements submit handler for devtrac_install_statistics_config_google_analytics_form.
- */
-function devtrac_install_statistics_config_google_analytics_form_submit($form, &$form_state) {
-  global $install_state;
-  $install_state['statistics']['googleanalytics']['googleanalytics_account'] = $form_state['values']['googleanalytics_account'];
-  variable_set('googleanalytics_account', $form_state['values']['googleanalytics_account']);
-}
-
 /**
  * Task callback: uses Batch API to import modules based on user selection.
  * Installs a site statistics module if requested, or any modules providing
