diff --git a/theme.inc b/theme.inc
index d990dd8..b451f4f 100644
--- a/theme.inc
+++ b/theme.inc
@@ -1,6 +1,11 @@
 <?php
 
 /**
+ * @file
+ * Controls the output of a views_two_column_table view.
+ */
+
+/**
  * Display the view as a two column table.
  */
 function template_preprocess_views_view_two_column_table(&$vars) {
diff --git a/views_plugin_style_two_column_table.inc b/views_plugin_style_two_column_table.inc
index 0a9f419..29759c7 100644
--- a/views_plugin_style_two_column_table.inc
+++ b/views_plugin_style_two_column_table.inc
@@ -10,7 +10,7 @@
  *
  * @ingroup views_style_plugins
  */
-class views_plugin_style_two column_table extends views_plugin_style_table {
+class views_plugin_style_two_column_table extends views_plugin_style_table {
   /**
    * Set default options
    */
diff --git a/views_two_column_table.info b/views_two_column_table.info
index 7428d05..4fa9355 100644
--- a/views_two_column_table.info
+++ b/views_two_column_table.info
@@ -2,4 +2,6 @@ name = Views Two Column Table
 description = Provides an alternate table row style for the Views module.
 dependencies[] = views
 package = Views
-core = 6.x
+core = 7.x
+
+files[] = views_plugin_style_two_column_table.inc
diff --git a/views_two_column_table.module b/views_two_column_table.module
index 80cd9f4..ee555ca 100644
--- a/views_two_column_table.module
+++ b/views_two_column_table.module
@@ -1,22 +1,22 @@
 <?php
 
 /**
- * Implementation of hook_views_api().
+ * Implements hook_views_api().
  */
 function views_two_column_table_views_api() {
   return array(
-    'api' => 2,
+    'api' => 3,
     'path' => drupal_get_path('module', 'views_two_column_table'),
   );
 }
 
 /**
- * Implementation of hook_views_plugins().
+ * Implements hook_views_plugins().
  */
 function views_two_column_table_views_plugins() {
   $path = drupal_get_path('module', 'views_two_column_table');
   $views_path = drupal_get_path('module', 'views');
-  require_once "./$path/theme.inc";
+  require_once DRUPAL_ROOT . '/' . "$path/theme.inc";
 
   $data = array(
     'module' => 'views_two_column_table',
@@ -37,7 +37,7 @@ function views_two_column_table_views_plugins() {
         'type' => 'normal',
         'even empty' => FALSE,
       ),
-    )
+    ),
   );
   return $data;
 }
